### DISCUSSION TITLE
Maya 2022 Webinar Featuring Maya USD, March 25th

### DISCUSSION THREAD

**Discussion by wtelford** [Autodesk/maya-usd#1276] [Announcement]
Now that Maya 2022 shipped today with the all new Maya USD plugin included, we will be hosting a free public webinar tomorrow
March 25th at 12:00-2:30 PM Eastern / 5:00-7:30 PM CET with a Q&A to follow. Please visit the following link for more details and to register for the event.

[Maya 2022 Webinar Featuring Maya USD](https://www.autodesk.com/campaigns/mande-events/maya-325)

I will be doing a deep dive of the features and workflows available in the plugin. I look forward to seeing you there.

Cheers,
Will Telford
[Twitter: @wtelford](https://twitter.com/wtelford)
Senior Product Owner, Maya USD

**Comment by Ennosch** [comment]
If you've missed it
https://www.youtube.com/watch?v=95h-3irNSCs

**Reply by wtelford** [reply]
You beat me to it! Thanks for posting!
